example/public/manifest.json

```diff
@@ -18,5 +18,28 @@
       "sizes": "862x568",
       "form_factor": "wide"
     }
+  ],
+  "icons": [
```

Comment:
What I was doing was automatic in the build process, is there any reason to have it manually? I don't like it that way but I'm open to change that.

Reply:
done by the assets plugin in the closeBundle hook